sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/configuration/SecHubConfigurationValidator.java

```diff
@@ -87,23 +87,30 @@ private void validateWebScan(SecHubConfiguration configuration, Errors errors) {
 		}
 
 		SecHubWebScanConfiguration webscan = webscanOption.get();
-		List<URI> uris = webscan.getUris();
-		if (uris.isEmpty()) {
+		URI uri = webscan.getUri();
+		
+		if (uri == null || uri.toString().isEmpty()) {
```

Comment:
When I first read `uri.toString().isEmpty()` my first thought was:  "???? why should a toString() call for a URI result on an empty string? at least schema and host must be there ... ?!?!??! "

But...
After trying out `URI uri = new URI("")` it became clear: this is possible in JRE - even if it makes absolutely no sense (IMHO).

But...
We have already made some other checks for `uri == null` (for example 
So we should introduce `SimpleNetworkUtils` (same location as `SimpleStringUtils`) and provide here a static method `isEmptyOrNull(uri)` or `isValidURI(uri)` and use this one everywhere - same implementation as yours in line 92 (at least in the changes done inside the PR and later in all other occurrences).




when an URI is not null it must be valid, so this check make no sense for me